common/utils/src/main/resources/error/error-classes.json

```diff
@@ -3030,6 +3030,60 @@
     ],
     "sqlState" : "42713"
   },
+  "STDS_COMMITTED_BATCH_UNAVAILABLE" : {
+    "message" : [
+      "No committed batch found, checkpoint location: <checkpointLocation>"
+    ],
+    "sqlState" : "KD000"
```

Comment:
Error classes are coming from specific data source - I don't know how each error class maps to the SQLSTATE, hence I marked all error classes to KD000, SQLSTATE specific to data source.

Reply:
cc @srielau